docs/components/concepts/process-instance-creation.md

```diff
@@ -123,22 +110,29 @@ Start instructions have the same [limitations as process instance modification](
 Start instructions are supported for both `CreateProcessInstance` commands.
 
 <details>
-  <summary>Code example</summary>
-  <p>
-Create a process instance starting before the 'ship_parcel' element:
-
-```java
-client.newCreateInstanceCommand()
-  .bpmnProcessId("order-process")
-  .latestVersion()
-  .variables(Map.of("orderId", "1234"))
-  .startBeforeElement("ship_parcel")
-  .send()
-  .join();
+   <summary>Create a process instance and await results via Camunda 8 REST API</summary>
```

Comment:
This does not wait for the instance completion, so this description is not correct. I believe we can copy the previous title for this (or stick close to that one at least).

Reply:
This was a copy-and-paste error, I think. 🙈